htdocs/js/notebook/notebook_model.js

```diff
@@ -303,6 +303,10 @@ Notebook.create_model = function()
                 select_range(Math.min(start, end), Math.max(start, end));
             }
 
+            _.each(this.cells, function(cell) {
+                cell.deactivate_cell();
+            });
+
```

Comment:
#1977 already covered blurring of cells when the selection changes, so this is unnecessary, removing it.